definitions/npm/password-validator_v4.x.x/flow_v0.134.x-/test_password-validator_v4.x.x.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import { describe, it } from 'flow-typed-test';
+import type { PasswordPolicySchema } from 'password-validator';
+import PasswordValidator from 'password-validator';
+
+describe('password-validator', () => {
+  it('should work with valid arguments', () => {
+    const schema: PasswordPolicySchema = new PasswordValidator();
+
+    // prettier-ignore
+    schema
+      .is().min(8)
+      .is().max(100)
+      .has().uppercase()
+      .has().lowercase()
+      .has().digits()
+      .has().symbols()
+      .has().not().spaces()
+      .has(/(.*[A-Z]){2,}/);
+
+    schema.validate('3L}Ls4[4}qQB>UgN');
+  });
+
+  it('should fail with incorrect arguments', () => {
+    const schema = new PasswordValidator();
+
+    // $FlowExpectedError
```

Comment:
You're missing error codes, check the failing tests.
E.g.: `$FlowExpectedError[extra-arg]`

Reply:
Updated.